docs/introduction.md

```diff
@@ -97,7 +97,7 @@ This isn't needed if your use case is feeding the \*arrs only. Since they have R
 
 As of right now, autobrr features:
 
-- Support for 75+ trackers with IRC announces
+- Support for 70+ trackers with IRC announces
```

Comment:
This is outdated information - revert the changes on this file.

Reply:
Strange. Had to manually change it, even after merging main into the branch. Probably because it was newer or something. Fixed.